Add internal documentation on deployments (#1021)
alphagov/govuk-prototype-kit: internal_docs/deploys.md

# Deploys

This repo uses [Heroku automatic deploys] to deploy the latest code in the
`main` branch directly to https://govuk-prototype-kit.herokuapp.com/.

> This is configured manually using the dashboard for the [govuk-prototype-kit

Reviewer: ```suggestion
Deployments are configured manually using the dashboard for the [govuk-prototype-kit
```

> pipeline]. Log into Heroku using the [team credentials].

Reviewer: Nice!
